realm/realm-library/src/objectServer/java/io/realm/RealmApp.java

```diff
@@ -124,7 +124,10 @@ private long init(RealmAppConfiguration config) {
                 config.getRequestTimeoutMs(),
                 syncDir,
                 userAgentBindingInfo,
-                appDefinedUserAgent);
+                appDefinedUserAgent,
+                "android",
```

Comment:
I used the description from here: https://docs.mongodb.com/stitch/users/user-activity/